code/extensions/che-commands/src/utils.ts

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************
+ * Copyright (c) 2023 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+/* eslint-disable header/header */
+
+import { V1alpha2DevWorkspaceSpecTemplateCommandsItemsExecEnv } from "@devfile/api";
+
+/*
+ * This function is used to resolve env variables in command line.
+ * It should support both syntaxes: ${VAR} and $VAR
```

Comment:
It should support or it really supports? )

Reply:
the answer is in tests 😉